### PR TITLE
Fixed a couple of typos

### PR DESCRIPTION
**Pull request by bryanajwells**
I spotted a couple of typos in the readme, so I have corrected them.

### CODE REVIEW

File: custom_components/solis/const.py [hunk 1]
```diff
@@ -34,8 +34,8 @@
 CONF_KEY_ID = 'portal_key_id'
 CONF_PLANT_ID = 'portal_plant_id'
 
-DOMAIN = "solis"
-SENSOR_PREFIX = 'Solis'
+DOMAIN = "solisRemmote"
+SENSOR_PREFIX = 'solisRemmote'
```
Comment (CorithMalin82) [inline custom_components/solis/const.py:38]:
```suggestion
DOMAIN = "solisRemote"
SENSOR_PREFIX = 'solisRemote'
```

File: custom_components/solis/manifest.json [hunk 2]
```diff
@@ -1,11 +1,11 @@
...
+  "domain": "solisRemote",
+  "version": "0.0.1",
   "config_flow": true,
-  "name": "Solis Inverter",
+  "name": "Solis Remote Control Inverter",
   "iot_class": "cloud_push",
-  "documentation": "https://github.com/hultenvp/solis-sensor/",
-  "issue_tracker": "https://github.com/hultenvp/solis-sensor/issues",
+  "documentation": "https://github.com/bryanajwells/solis-sensor",
+  "issue_tracker": "https://github.com/bryanajwells/solis-sensor/issues",
```
Comment (CorithMalin82) [inline custom_components/solis/manifest.json:8]: Did you mean to leave these in?  It seems they would only apply to your fork?

File: custom_components/solis/manifest.json [hunk 3]
```diff
@@ -1,11 +1,11 @@
...
   "dependencies": [],
-  "codeowners": ["@hultenvp"]
+  "codeowners": ["@bryanajwells"]
```
Comment (CorithMalin82) [inline custom_components/solis/manifest.json:10]: Did you mean to leave these in?  It seems they would only apply to your fork?

**Review by CorithMalin82 — COMMENTED** [review]
Just helping with a second pair of eyes.  Great stuff!

File: custom_components/solis/manifest.json (hunk 2 above)
Comment (bryanajwells) [reply custom_components/solis/manifest.json:8]: Good spot, a bit new to github.  Will re-prepare.

File: custom_components/solis/manifest.json (hunk 3 above)
Comment (bryanajwells) [reply custom_components/solis/manifest.json:10]: Good spot, a bit new to github.  Will re-prepare.

**Comment by bryanajwells** [comment]
Ah I see what's happened. I've wronly assumed that the pull requst would pin to the point where I set up the request, it doesn't, and its picked up the subsequent changes.

**Comment by bryanajwells** [comment]
Replaced with a differnet review.

**Pull request closed without merging**